source/administration/object-management/object-versioning.rst

```diff
@@ -266,6 +265,129 @@ enable versioning on only a prefix or subset of objects in a bucket.
 Objects created prior to enabling versioning have a 
 ``null`` :ref:`version ID <minio-bucket-versioning-id>`.
 
+
+Exclude a Prefix From Versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can exclude certain :ref:`prefixes <minio-admin-concepts-organize-objects>` from versioning using the :ref:`MinIO Client <minio-client>`.
+This is useful for Spark/Hadoop workloads or others that initially create objects with temporary prefixes. 
+
+.. admonition:: Object locking
+   :class: note
+
+   Buckets with :ref:`object locking enabled <minio-object-locking>` require versioning and do not support excluding prefixes.
+
+- Use :mc-cmd:`mc version enable` with the :mc-cmd:`~mc version --excluded-prefixes` option:
+
+  .. code-block:: shell
+     :class: copyable
+
+     mc version enable --excluded-prefixes "prefix1, prefix2" ALIAS/BUCKET
+
+  - Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+  - Replace ``BUCKET`` with the name of the :s3-docs:`bucket <UsingBucket.html>` you want to exclude :ref:`prefixes <minio-admin-concepts-organize-objects>` for.
+
+The list of :mc-cmd:`~mc version --excluded-prefixes` prefixes match all objects containing the specified strings in their prefix or name, similar to a regular expression of the form ``*prefix*``.
```

Comment:
In this example, the given excluded prefixes `prefix1`, `prefix2` would match objects similar to the regular expression of the form `prefix*`.
```suggestion
The list of :mc-cmd:`~mc version --excluded-prefixes` prefixes match all objects containing the specified strings in their prefix or name, similar to a regular expression of the form ``prefix*``.
```